packages/merkle-distributor/.d.ts

```diff
@@ -0,0 +1 @@
+declare module "ethereumjs-util";
```

Comment:
Had to add this file because of the error when trying to build `merkle-distributor`:

```
@uma/merkle-distributor: src/MerkleTree.ts(3,40): error TS7016: Could not find a declaration file for module 'ethereumjs-util'. '/home/circleci/protocol/node_modules/ethereumjs-util/dist/index.js' implicitly has an 'any' type.
@uma/merkle-distributor:   Try `npm i --save-dev @types/ethereumjs-util` if it exists or add a new declaration (.d.ts) file containing `declare module 'ethereumjs-util';`
@uma/merkle-distributor: error Command failed with exit code 1.
@uma/merkle-distributor: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```